server/routes/index.js

```diff
@@ -49,4 +50,5 @@ export default (app) => {
   app.use('/api/v1/assign-badge', assignedBadge);
   app.use('/api/v1/bank-account', bankAccount);
   app.use('/api/v1/property-video', propertyVideo);
+  app.use('/api/v1/value-added-service', valueAddedService);
```

Comment:
As discussed above, I think we can use `vas` and it would still make complete sense.
```suggestion
  app.use('/api/v1/vas', valueAddedService);
```